infra/gcp/roles/access-logs.viewer.yaml

```diff
@@ -0,0 +1,24 @@
+#### generated by generate-role-yaml.sh from specs/access-logs.viewer.yaml
```

Comment:
I don't believe this comment.  When I ran `./roles/generate-role-yaml.sh` I saw significantly more permissions show up

Reply:
That's odd. I had to modify the script to get it to work locally
```diff
diff --git a/infra/gcp/roles/generate-role-yaml.sh b/infra/gcp/roles/generate-role-yaml.sh
index 7f5c8aae..4eba7280 100755
--- a/infra/gcp/roles/generate-role-yaml.sh
+++ b/infra/gcp/roles/generate-role-yaml.sh
@@ -68,13 +68,13 @@ function output_role_yaml() {
   local spec="${1}"
 
   local title description name include_roles include_permissions include_regex exclude_regex
-  title=$(<"${spec}" yq -r .title)
-  description=$(<"${spec}" yq -r .description)
-  name=$(<"${spec}" yq -r .name)
-  mapfile -t include_roles < <(<"${spec}" yq -r '.include? | .roles//[] | .[]')
-  mapfile -t include_permissions < <(<"${spec}" yq -r '.include? | .permissions//[] | .[]')
-  include_regex=$(<"${spec}" yq -r '.include? | .permissionRegexes//[] | join("|")')
-  exclude_regex=$(<"${spec}" yq -r '.exclude? | .permissionRegexes//[] | join("|")')
+  title=$(<"${spec}" yq e -M .title -)
+  description=$(<"${spec}" yq e -M .description -)
+  name=$(<"${spec}" yq e -M .name -)
+  mapfile -t include_roles < <(<"${spec}" yq e -M '.include? | .roles//[] | .[]' -)
+  mapfile -t include_permissions < <(<"${spec}" yq e -M '.include? | .permissions//[] | .[]' -)
+  include_regex=$(<"${spec}" yq e -M '.include? | .permissionRegexes//[] | join("|")' -)
+  exclude_regex=$(<"${spec}" yq e -M '.exclude? | .permissionRegexes//[] | join("|")' -)
 
   local output_path="${output_dir}/${name}.yaml"
```

I'm using [mikefarah/yq](https://github.com/mikefarah/yq), not [Python yq](https://pypi.org/project/yq/). To reproduce, the last commit before mine is 5de37c3784916ec2115adfbb3513519cd317fb4a. I think I must've broken something